CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add `supportDownlinkBandwidthEstimation` API to check whether browsers support downlink bandwidth estimation 
   which requires for priority based downlink policy to work.
 - Add `keepLastFrameWhenPaused` in `DefaultVideoTile` as an option to keep last frame when pausing a video tile.
+- Add support to send and receive stereo audio
```

Comment:
Can we add a bit about the use case that builders would want to use this here?

Reply:
Updated changelog. Also added demo updates.